client/platform/desktop/backend/ipcService.ts

```diff
@@ -1,9 +1,10 @@
 import { ipcMain } from 'electron';
-
+import OS from 'os';
```

Comment:
language imports should go above external imports.

Need to find the linter settings for that...